client/src/i18n/en/form.json

```diff
@@ -774,7 +774,7 @@
             "EXCEPTION": "There was a problem validating this field. Please check your internet connection.",
             "INTEGER": "This value must be a whole number (integer).",
             "INVALID_DATE": "Invalid date",
-            "MAX": "Maximum value not respected",
+            "MAX": "This value is too large!  Choose a lower value.",
```

Comment:
I think this is much clearer feedback for the user! Could this be applied to `fr/form.json` also? 

Current value:
```
"MAX": "Valeur maximale non respectée"
```